chain/network/src/test_utils.rs

```diff
@@ -411,7 +411,6 @@ impl PeerManagerAdapter for NetworkRecipient {
 
 #[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
 #[derive(Message, Clone, Debug)]
-#[cfg(feature = "test_features")]
```

Comment:
Because conditional compilation is evil and our code is overusing it a lot. What is the point of hiding pieces of code from the compiler within the test suite? The only effect I observe is that it is next to impossible to tell whether our code compiles under every feature flag combination.